rest/making-calls-sip/example-1/example-1.5.x.cs

```diff
@@ -0,0 +1,23 @@
+// Download the twilio-csharp library from twilio.com/docs/csharp/install
+using System;
+using Twilio;
+using Twilio.Rest.Api.V2010.Account;
+using Twilio.Types;
+
+class Example
+{
+  static void Main(string[] args)
+  {
+    // Find your Account Sid and Auth Token at twilio.com/console
+    const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
+    const string authToken = "your_auth_token";
+    TwilioClient.Init(accountSid, authToken);
+
+    var url = new Uri("http://www.example.com/sipdial.xml");
```

Comment:
No need for the extra variable `url`, you can use directly on `CallResource.Create` as you use a named parameter the semantic value is provided by the parameter name.